sitzverteilung-frontend/src/plugins/router.ts

```
@@ -1,11 +1,13 @@
 import { createRouter, createWebHashHistory } from "vue-router";
 
 import AboutView from "@/views/AboutView.vue";
+import CalculationView from "@/views/CalculationView.vue";
 import DataView from "@/views/DataView.vue";
```

Biome (2.1.2)

[error] Do not shadow the global "DataView" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)